dhis-2/dhis-services/dhis-service-setting/src/main/java/org/hisp/dhis/setting/SettingKey.java

```diff
@@ -159,8 +158,8 @@
     SYSTEM_METADATA_VERSION( "keySystemMetadataVersion", String.class ),
     STOP_METADATA_SYNC( "keyStopMetadataSync", Boolean.FALSE, Boolean.class ),
     FILE_RESOURCE_RETENTION_STRATEGY( "keyFileResourceRetentionStrategy", FileResourceRetentionStrategy.NONE, FileResourceRetentionStrategy.class ),
-    TRACKER_SYNC_PAGE_SIZE( "syncTrackerPageSize", 20, Integer.class ),
-    EVENT_SYNC_PAGE_SIZE( "syncEventsPageSize", 60, Integer.class ),
+    TRACKER_PROGRAM_SYNC_PAGE_SIZE( "syncTrackerPageSize", 20, Integer.class ),
```

Comment:
Would it make sense to have these values as part of the jobConfiguration, not systemSettings?

Reply:
Could be an option. But then we would have to add job parameters also to the Metadata sync job (and here 3 page sizes will have to be specified: tracker, events, and completeness (coming soon) ). Also, this setting is more for performance tuning. Maybe people won't change it at all. Or won't change until their sync jobs will be crashing/timing out too often.

Reply:
I think thats fine. When we have jobparamters, I dont feel like these values makes sense to put in system settings, as they are actually part of a specific job. You can set the default value of the jobParameter to be the defaults you have here, then people can tune if they need to :)

Reply:
OK. Does it make sense to you to merge this PR and I will change the usage of SystemSettings to JobParameters in a new one? If yes, then please merge and I will create a new PR with the mentioned change.

Reply:
Sure I can do that.